### ISSUE TITLE
[dataflows] revisit loading multiple sheets with one load step

### ISSUE THREAD

**Issue by cschloer** [frictionlessdata/datapackage-pipelines#188]
see https://github.com/datahq/dataflows/pull/110

@roll

**Comment by roll** [comment]
Thanks @cschloer 

I'll take a look

**Comment by roll** [comment]
Hi @cschloer,

> The only thing left is integrating goodtables into dataflows/datapackage pipelines.

Does this mean that this issue is not needed anymore?

**Comment by cschloer** [comment]
This would still be great, but as I said in the linked issue we already have the "batch load" concept in our custom load step. And I guess since you said you didn't think the `tabulator.Collections` solution would be possible in this stage of the pilot, maybe it makes sense to remove it?

**Comment by roll** [comment]
Yes, I think it's better to remove it from this stage. I think in a few months it will be ready so you will be able to rebase from your custom processor
